client/components/TestRenderer/index.jsx

```diff
@@ -217,100 +218,182 @@ const ErrorComponent = ({ hasErrors = false }) => {
 };
 
 const TestRenderer = ({
-    test,
-    testPageUri,
-    support,
-    configQueryParams = [[]],
+    at,
+    testResult = {},
+    testPageUrl,
     testRunStateRef,
     testRunResultRef,
     submitButtonRef,
     isSubmitted = false
 }) => {
-    const { title, testJson, commandJson, scripts, state, result } = test;
+    const { scenarioResults, test = {}, completedAt } = testResult;
+    const { renderableContent } = test;
 
+    const [testRunExport, setTestRunExport] = useState();
     const [pageContent, setPageContent] = useState(null);
-    const [submitResult, setSubmitResult] = useState(result);
-
-    /**
-     * Script tag parsed from test page html can contain structure like:
-     * @example
-     * var scripts = {
-     *   <func>: function(...) {
-     *     // ...
-     *   }
-     * };
-     *
-     * where <func> is a setup script intended to be ran when the test window
-     * is triggered.
-     *
-     * This is constructing the 'scripts' STRING variable as an internal object,
-     * '_scripts' if there are an available 'scripts' variable using eval.
-     */
-    let _scripts = null;
-    const _scriptsString = scripts.replace('var scripts', '_scripts');
-    eval(_scriptsString);
-
-    const testRunIO = new TestRunInputOutput();
-    testRunIO.setTitleInputFromTitle(title);
-    testRunIO.setUnexpectedInputFromBuiltin();
-    testRunIO.setScriptsInputFromMap(
-        typeof _scripts === 'object' ? _scripts : {}
-    );
-    testRunIO.setSupportInputFromJSON(support);
-    testRunIO.setConfigInputFromQueryParamsAndSupport(configQueryParams); // Array.from(new URL(document.location).searchParams)
-    testRunIO.setKeysInputFromBuiltinAndConfig();
-    testRunIO.setCommandsInputFromJSONAndConfigKeys(commandJson);
-    testRunIO.setBehaviorInputFromJSONAndCommandsConfigKeysTitleUnexpected(
-        testJson
-    );
-    testRunIO.setPageUriInputFromPageUri(testPageUri);
+    const [state, setState] = useState(null);
```

Comment:
This variable name seems quite generic, something like testRendererState might work better? I do think whatever you choose should extend to the other variables like `_state`.